### PR TITLE
feat: Fix some of the linter errors

### PR DESCRIPTION
**Pull request by eriksvedang**
The linter is still turned off in the github action.

### CODE REVIEW

**Comment by eriksvedang** [comment]
A lot of these fixes are not very good, but at least it brings us closer to being able to turn on the linter again (so that it can find actual problems...)

File: src/connection/server_logic.go [hunk 1]
```diff
@@ -193,9 +193,11 @@ func (s *ServerLogic) OnConnectRequest(addr *endpoint.Endpoint, requestMessage *
 		if setErr != nil {
 			return setErr
 		}
-	} else {
-		//return fmt.Errorf("We already have a connection for nonce: %d", requestMessage.ClientDeterminedNonce)
 	}
+	// else {
+	//return fmt.Errorf("We already have a connection for nonce: %d", requestMessage.ClientDeterminedNonce)
+	//}
+
```
Comment (rpadrela) [inline src/connection/server_logic.go:200]: remove if not needed

Comment (eriksvedang) [reply src/connection/server_logic.go:200]: 👍

**Review by rpadrela — CHANGES_REQUESTED** [review]
There's some commented out code being submitted. Either uncomment or remove it.

**Comment by eriksvedang** [comment]
Done!

**Review by rpadrela — APPROVED** [review]
Good stuff.

**Pull request merged**